packages/devtools_app/lib/src/network/network_model.dart

```diff
@@ -83,6 +85,10 @@ class WebSocket extends NetworkRequest {
   @override
   int get port => _socket.port;
 
+  // TODO(kenz): what determines a web socket request failure?
+  @override
+  bool get didFail => false;
```

Comment:
@bkonyi do we get information from web socket requests that signal success or failure? I'm not seeing anything that would tell us this in the `SocketStatistic` class

Reply:
I don't think the socket profiling covers `WebSocket`, just the `dart:io` `Socket` classes.

Reply:
The WS examples in the screenshot are created via `IOWebSocketChannel.connect('ws://echo.websocket.org').sink.add('example message')`, and those are pulled from the socket profile.

Reply:
Oh I thought they were pulled from the HTTP profiler. I'll have to take a look at the web socket implementation first, but I'll get back to you on this. I'm 99% sure the web socket implementation is built on top of `Socket` at some level, so I doubt there's any special information about success / failure for web sockets that isn't already available for sockets.